R/routes.R

```diff
@@ -80,7 +80,7 @@
 #' route_cyclestreet(c(-2, 52), c(-1, 53), "fastest")
 #' }
 route_cyclestreet <- function(from, to, plan = "fastest", silent = TRUE, pat = cyclestreet_pat(),
-                              base_url = "http://www.cyclestreets.net/api/", reporterrors = FALSE){
+                              base_url = "http://www.cyclestreets.net", reporterrors = FALSE){
```

Comment:
Just to clarify here, the base url hasn't changed. I've just appended the path in using the `path` argument in [`httr::modify_url()`](https://github.com/hadley/httr/blob/master/R/url.r#L152):

https://github.com/ropensci/stplanr/pull/119/files#diff-492212fe1d20a03daa2020e2bcfc2ea0R103

I think it makes sense to allow users to set-up their own base url, it's what we've done with the osrm functions. Great resource that led me to `modify_url()` btw: https://cran.r-project.org/web/packages/httr/vignettes/api-packages.html